packages/scene-composer/src/components/Tree/tree.scss

```diff
@@ -42,6 +42,11 @@ $active-color: #ec7211;
     position: relative;
     user-select: none;
 
+    &:first-child {
```

Comment:
with https://github.com/awslabs/iot-app-kit/pull/411 you should only need this css change